crate_specific.nix

```diff
@@ -0,0 +1,16 @@
+# This file contains crate-specific build inputs.
+#
+# Each of them is an attribute with name of the crate and value being a function.
+# This function gets as arguments:
+#  - `crateInfo` with information about the crate, such as sha256 or version
+#
+# Currently supported fields to return are: `buildInputs`, `nativeBuildInputs`
```

Comment:
And the returned `buildInputs` are simply added to the other build inputs, right?

Reply:
Yes, here:
https://github.com/nix-community/naersk/pull/326/files#diff-b6b537316f2d29c8faf178a110366796811d1bc72f694262c7d2efad79aa984bR146